cmd/socialtokenrefresh/socialtokenrefresh.go

```diff
@@ -105,6 +107,13 @@ func initWithEnv() error {
 		return fmt.Errorf("SocialTokenRefresh [Init Firestore]: %v", err)
 	}
 
+	// Initialize Sawmill
+	sawmillLogger, err := sawmill.InitClient(currentProject, os.Getenv("GCLOUD_CONFIG"), os.Getenv("ENVIRONMENT"), "SocialTokenRefersh")
+	if err != nil {
+		log.Printf("SocialTokenRefresh [Init Sawmill]: %v", err)
+	}
+
+	logger = sawmillLogger
 	firestoreClient = client
```

Comment:
You remove firestoreClient? Or am I reading this wrong?

Reply:
I think I just populated `sawmillClient` before `firestoreClient`. I'll fix that order.